integrationExamples/videoModule/videojsAdapter/index.html

```diff
@@ -0,0 +1,105 @@
+<html>
+
+<head>
+    <script async src="../../../build/dev/prebid.js"></script>
+    <link href="https://vjs.zencdn.net/7.17.0/video-js.css" rel="stylesheet" />
+    <!-- Setup ad units -->
+    <script>
+      var adUnits = [{
+        code: 'div-gpt-ad-51545-0',
+        sizes: [[600, 500]],
+        mediaTypes: {
+          video: {
+            sizes: [[600, 500]]
+          }
+        },
+        video: {
+          divId: 'vid-div-1',
+          adServer: {
+            vendorCode: "gam",
+            params: {
+              iu: '/19968336/prebid_cache_video_adunit',
+              cust_params: {
+                section: 'blog',
+                anotherKey: 'anotherValue'
+              },
+              output: 'vast'
+            }
+          },
+        },
+
+        bids: [{
+          bidder: 'ix',
+          params: {
+            siteId: '300',
+            size: [600, 500]
+          }
+        }]
+      }];
+
+      var pbjs = pbjs || {};
+      pbjs.que = pbjs.que || [];
+
+      pbjs.que.push(function () {
+        pbjs.onEvent('adImpression', event => {
+          console.log('adImpression: ', event);
+        });
+        pbjs.addAdUnits(adUnits);
+        pbjs.setConfig({
```

Comment:
WeI think in general it's best to call `setConfig` before `addAdUnits` since a lot of module setup happens in `setConfig`. If it's the same in the jwplayer demo then we'll want to update it as well. 
Functionality wise it might not matter but since developers will use these pages as documentation, we don't want to confuse them